lib/blackjack_rb/game.rb

```diff
@@ -24,31 +21,88 @@ def initialize(number_of_players)
     # after the players are done, go do the dealer actions.
     # when the dealer has finished playing, go through a score round
     def play
-      give_initial_cards
-
+      SHELL.confirm("Let's play some blackjack!")
       loop do
-        # GET BETS FROM PLAYERS HERE
+        collect_bets
+        give_initial_cards
+        # ask if player would like to split here
         @players.each { |player| action_cycle(player, :player) }
-        # action_cycle(@dealer, :dealer)
-        # decide_scores
+        action_cycle(@dealer, :dealer)
+        decide_scores
         break if done_playing?
+        reset
       end
     end
 
-    def action_cycle(person, who)
+    def collect_bets
+      @players.each { |player| player.collect_bet }
+    end
+
+    def action_cycle(player, who)
       response = nil
       loop do
         if who.eql?(:player)
-          puts "YOUR CARDS \n#{person.hand.cards.inspect}"
-          puts "DEALER'S CARD \n#{@dealer.hand.cards[0].inspect}"
-          # RENDER PLAYER CARDS HERE
-          # RENDER DEALER CARDS
+          RENDERER.render(@dealer.hand.cards, :dealer)
+          RENDERER.render(player.hand.cards, who)
+
           SHELL.confirm("Would you like to hit or stay or double?")
           response = SHELL.ask("Enter 's' to stay, 'h' to hit, or 'd' to double.").argument(:required).on_error(:retry).valid(['s', 'h', 'd']).read_char
-          person.receive(@deck.draw_card) if response.eql?('h')
-          # ask if player would like to split here
+
+          double_player(player) if response.eql?('d')
+          hit_player(player) if (response.eql?('h') || response.eql?('d'))
```

Comment:
Don't use parentheses around the condition of an `if`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.